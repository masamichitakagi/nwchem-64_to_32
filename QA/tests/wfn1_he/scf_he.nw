echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
  * library 6-31g*
end

dft
  odft
  mult 1
  smear 0.0
  xc hfexch
end

task dft energy

dft
  odft
  mult 1
  smear 1.5
  xc hfexch
end

task dft energy
