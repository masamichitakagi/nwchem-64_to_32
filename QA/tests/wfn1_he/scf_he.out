 argument  1 = scf_he.nw



============================== echo of input deck ==============================
echo
start wfn1_he_dat

geometry
  he 0 0 0
end
basis
  * library 6-31g*
end

dft
  odft
  mult 1
  smear 0.0
  xc hfexch
end

task dft energy

dft
  odft
  mult 1
  smear 1.5
  xc hfexch
end

task dft energy
================================================================================


                                         
                                         


              Northwest Computational Chemistry Package (NWChem) 6.6
              ------------------------------------------------------


                    Environmental Molecular Sciences Laboratory
                       Pacific Northwest National Laboratory
                                Richland, WA 99352

                              Copyright (c) 1994-2015
                       Pacific Northwest National Laboratory
                            Battelle Memorial Institute

             NWChem is an open-source computational chemistry package
                        distributed under the terms of the
                      Educational Community License (ECL) 2.0
             A copy of the license is included with this distribution
                              in the LICENSE.TXT file

                                  ACKNOWLEDGMENT
                                  --------------

            This software and its documentation were developed at the
            EMSL at Pacific Northwest National Laboratory, a multiprogram
            national laboratory, operated for the U.S. Department of Energy
            by Battelle under Contract Number DE-AC05-76RL01830. Support
            for this work was provided by the Department of Energy Office
            of Biological and Environmental Research, Office of Basic
            Energy Sciences, and the Office of Advanced Scientific Computing.


           Job information
           ---------------

    hostname        = 10-4-18-102.user.vpn.bnl.local
    program         = ../../../bin/MACX64/nwchem
    date            = Sun Jan  8 23:16:10 2017

    compiled        = Sun_Jan_08_22:47:59_2017
    source          = /Users/vandam/nwchem-wfn1-dev
    nwchem branch   = Development
    nwchem revision = 28658
    ga revision     = 10724
    input           = scf_he.nw
    prefix          = wfn1_he_dat.
    data base       = ./wfn1_he_dat.db
    status          = startup
    nproc           =        1
    time left       =     -1s



           Memory information
           ------------------

    heap     =   13107196 doubles =    100.0 Mbytes
    stack    =   13107201 doubles =    100.0 Mbytes
    global   =   26214400 doubles =    200.0 Mbytes (distinct from heap & stack)
    total    =   52428797 doubles =    400.0 Mbytes
    verify   = yes
    hardfail = no 


           Directory information
           ---------------------

  0 permanent = .
  0 scratch   = .




                                NWChem Input Module
                                -------------------



 Scaling coordinates for geometry "geometry" by  1.889725989
 (inverse scale =  0.529177249)



                             Geometry "geometry" -> ""
                             -------------------------

 Output coordinates in angstroms (scale by  1.889725989 to convert to a.u.)

  No.       Tag          Charge          X              Y              Z
 ---- ---------------- ---------- -------------- -------------- --------------
    1 he                   2.0000     0.00000000     0.00000000     0.00000000

      Atomic Mass 
      ----------- 

      he                 4.002600


 Effective nuclear repulsion energy (a.u.)       0.0000000000

            Nuclear Dipole moment (a.u.) 
            ----------------------------
        X                 Y               Z
 ---------------- ---------------- ----------------
     0.0000000000     0.0000000000     0.0000000000


            XYZ format geometry
            -------------------
     1
 geometry
 he                    0.00000000     0.00000000     0.00000000



 Summary of "ao basis" -> "" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 *                           6-31g*                   on all atoms 


  xc_inp: hfexch multiplicative factor not found.

                                 NWChem DFT Module
                                 -----------------


                      Basis "ao basis" -> "ao basis" (cartesian)
                      -----
  he (Helium)
  -----------
            Exponent  Coefficients 
       -------------- ---------------------------------------------------------
  1 S  3.84216340E+01  0.023766
  1 S  5.77803000E+00  0.154679
  1 S  1.24177400E+00  0.469630

  2 S  2.97964000E-01  1.000000



 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                          6-31g*                  2        2   2s




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                          6-31g*                  2        2   2s


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     1
            Beta electrons :     1
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:     2
                     number of shells:     2
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

          Smearing applied:  0.00D+00 (hartree)

      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


      Superposition of Atomic Density Guess
      -------------------------------------

 Sum of atomic energies:          -2.85516040

      Non-variational initial energy
      ------------------------------

 Total energy =      -2.855160
 1-e energy   =      -3.882067
 2-e energy   =       1.026907
 HOMO         =      -0.914127
 LUMO         =       1.399859

   Time after variat. SCF:      0.1
   Time prior to 1st pass:      0.1

 #quartets = 6.000D+00 #integrals = 6.000D+00 #direct =  0.0% #cached =100.0%


 Integral file          = ./wfn1_he_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  57138
 No. of bits per label  =      8        No. of bits per value  =     64


           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.98            12975588
          Stack Space remaining (MW):       13.11            13107068

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -2.8551604262 -2.86D+00  4.83D-09  2.97D-16     0.2
                                                     4.83D-09  2.97D-16
  Resetting Diis
 d= 0,ls=0.0,diis     2     -2.8551604262  4.44D-16  3.95D-10  1.70D-18     0.2
                                                     3.95D-10  1.70D-18


         Total DFT energy =       -2.855160426154
      One electron energy =       -3.882067594485
           Coulomb energy =        2.053814336663
    Exchange-Corr. energy =       -1.026907168331
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.1s



                    DFT Final Alpha Molecular Orbital Analysis
                    ------------------------------------------

 Vector    1  Occ=1.000000D+00  E=-9.141266D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.592081  1 He s                  2      0.513586  1 He s          

 Vector    2  Occ=0.000000D+00  E= 1.399859D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.186959  1 He s                  1      1.149818  1 He s          


                     DFT Final Beta Molecular Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=1.000000D+00  E=-9.141266D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.592081  1 He s                  2      0.513586  1 He s          

 Vector    2  Occ=0.000000D+00  E= 1.399859D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.186959  1 He s                  1      1.149818  1 He s          


   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2
    beta      1      2
 overlap   1.000  1.000

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =     -0.0000 (Exact =     0.0000)


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.2s     wall:        0.2s


                                NWChem Input Module
                                -------------------


  xc_inp: hfexch multiplicative factor not found.

                                 NWChem DFT Module
                                 -----------------




 Summary of "ao basis" -> "ao basis" (cartesian)
 ------------------------------------------------------------------------------
       Tag                 Description            Shells   Functions and Types
 ---------------- ------------------------------  ------  ---------------------
 he                          6-31g*                  2        2   2s


  Caching 1-el integrals 

            General Information
            -------------------
          SCF calculation type: DFT
          Wavefunction type:  spin polarized.
          No. of atoms     :     1
          No. of electrons :     2
           Alpha electrons :     1
            Beta electrons :     1
          Charge           :     0
          Spin multiplicity:     1
          Use of symmetry is: off; symmetry adaption is: off
          Maximum number of iterations:  30
          AO basis - number of functions:     2
                     number of shells:     2
          Convergence on energy requested:  1.00D-06
          Convergence on density requested:  1.00D-05
          Convergence on gradient requested:  5.00D-04

              XC Information
              --------------
                     Hartree-Fock (Exact) Exchange  1.000          

          Convergence Information
          -----------------------
          Convergence aids based upon iterative change in 
          total energy or number of iterations. 
          Levelshifting, if invoked, occurs when the 
          HOMO/LUMO gap drops below (HL_TOL):  1.00D-02
          DIIS, if invoked, will attempt to extrapolate 
          using up to (NFOCK): 10 stored Fock matrices.

                    Damping( 0%)  Levelshifting(0.5)       DIIS
                  --------------- ------------------- ---------------
          dE  on:    start            ASAP                start   
          dE off:    2 iters         30 iters            30 iters 

          Smearing applied:  0.15D+01 (hartree)

      Screening Tolerance Information
      -------------------------------
          Density screening/tol_rho:  1.00D-10
          AO Gaussian exp screening on grid/accAOfunc:  14
          CD Gaussian exp screening on grid/accCDfunc:  20
          XC Gaussian exp screening on grid/accXCfunc:  20
          Schwarz screening/accCoul:  1.00D-08


 Loading old vectors from job with title :



   Time after variat. SCF:      0.3
 sigma=  0.15D+01 eFermi=  0.00000 T*S= -9.38E-01 nel(1)=    1.00 nel(2)=    1.00

   Time prior to 1st pass:      0.3

 #quartets = 6.000D+00 #integrals = 6.000D+00 #direct =  0.0% #cached =100.0%


 Integral file          = ./wfn1_he_dat.aoints.0
 Record size in doubles =  65536        No. of integs per rec  =  43688
 Max. records in memory =      2        Max. records in file   =  57138
 No. of bits per label  =      8        No. of bits per value  =     64

 sigma=  0.15D+01 eFermi=  0.24287 T*S= -9.34E-01 nel(1)=    1.00 nel(2)=    1.00

  Resetting Diis

           Memory utilization after 1st SCF pass: 
           Heap Space remaining (MW):       12.98            12975668
          Stack Space remaining (MW):       13.11            13107036

   convergence    iter        energy       DeltaE   RMS-Dens  Diis-err    time
 ---------------- ----- ----------------- --------- --------- ---------  ------
 d= 0,ls=0.0,diis     1     -3.7887666349 -3.79D+00  1.93D-01  1.04D-29     0.3
                                                     1.93D-01  1.42D-29
 sigma=  0.15D+01 eFermi=  0.22493 T*S= -1.01E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     2     -3.2526498487  5.36D-01  2.49D-02  7.39D-05     0.4
                                                     2.49D-02  7.39D-05
 sigma=  0.15D+01 eFermi=  0.22400 T*S= -1.02E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     3     -3.1898688743  6.28D-02  4.19D-03  2.52D-06     0.4
                                                     4.19D-03  2.52D-06
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 sigma=  0.15D+01 eFermi=  0.22415 T*S= -1.02E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     4     -3.1767042150  1.32D-02  2.22D-04  4.30D-09     0.4
                                                     2.22D-04  4.30D-09
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 sigma=  0.15D+01 eFermi=  0.22415 T*S= -1.02E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     5     -3.1773440223 -6.40D-04  8.85D-06  6.34D-12     0.5
                                                     8.85D-06  6.34D-12
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 sigma=  0.15D+01 eFermi=  0.22415 T*S= -1.02E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     6     -3.1773705909 -2.66D-05  2.19D-09  3.89D-19     0.5
                                                     2.19D-09  3.89D-19
  Singularity in Pulay matrix. Error and Fock matrices removed. 
  Singularity in Pulay matrix. Error and Fock matrices removed. 
 sigma=  0.15D+01 eFermi=  0.22415 T*S= -1.02E+00 nel(1)=    1.00 nel(2)=    1.00

 d= 0,ls=0.0,diis     7     -3.1773705844  6.57D-09  1.62D-13  2.52D-27     0.6
                                                     1.62D-13  2.49D-27


         Total DFT energy =       -2.153298259089
      One electron energy =       -3.300611603530
           Coulomb energy =        1.959023734127
    Exchange-Corr. energy =       -0.811710389686
 Nuclear repulsion energy =        0.000000000000

 Numeric. integr. density =        0.000000000000

     Total iterative time =      0.3s



                    DFT Final Alpha Molecular Orbital Analysis
                    ------------------------------------------

 Vector    1  Occ=8.419386D-01  E=-8.391153D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.597594  1 He s                  2      0.507882  1 He s          

 Vector    2  Occ=1.580614D-01  E= 1.287418D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.189410  1 He s                  1      1.146963  1 He s          


                     DFT Final Beta Molecular Orbital Analysis
                     -----------------------------------------

 Vector    1  Occ=8.419386D-01  E=-8.391153D-01
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 3.2D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     1      0.597594  1 He s                  2      0.507882  1 He s          

 Vector    2  Occ=1.580614D-01  E= 1.287418D+00
              MO Center=  0.0D+00,  0.0D+00,  0.0D+00, r^2= 7.3D-01
   Bfn.  Coefficient  Atom+Function         Bfn.  Coefficient  Atom+Function  
  ----- ------------  ---------------      ----- ------------  ---------------
     2     -1.189410  1 He s                  1      1.146963  1 He s          


   alpha - beta orbital overlaps 
   ----------------------------- 


   alpha      1      2
    beta      1      2
 overlap   1.000  1.000

     --------------------------
     Expectation value of S2:  
     --------------------------
      <S2> =      0.2662 (Exact =     0.0000)


 Parallel integral file used       1 records with       0 large values


 Task  times  cpu:        0.4s     wall:        0.4s


                                NWChem Input Module
                                -------------------


 Summary of allocated global arrays
-----------------------------------
  No active global arrays



                         GA Statistics for process    0
                         ------------------------------

       create   destroy   get      put      acc     scatter   gather  read&inc
calls:  363      363      842      428      226        0        0      110     
number of processes/call 1.00e+00 1.00e+00 1.00e+00 0.00e+00 0.00e+00
bytes total:             2.48e+04 1.19e+04 7.23e+03 0.00e+00 0.00e+00 8.80e+02
bytes remote:            0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00 0.00e+00
Max memory consumed for GA by this process: 2080 bytes

MA_summarize_allocated_blocks: starting scan ...
MA_summarize_allocated_blocks: scan completed: 0 heap blocks, 0 stack blocks
MA usage statistics:

	allocation statistics:
					      heap	     stack
					      ----	     -----
	current number of blocks	         0	         0
	maximum number of blocks	        19	        23
	current total bytes		         0	         0
	maximum total bytes		   1052888	  22509672
	maximum total K-bytes		      1053	     22510
	maximum total M-bytes		         2	        23


                                     CITATION
                                     --------
                Please cite the following reference when publishing
                           results obtained with NWChem:

                 M. Valiev, E.J. Bylaska, N. Govind, K. Kowalski,
              T.P. Straatsma, H.J.J. van Dam, D. Wang, J. Nieplocha,
                        E. Apra, T.L. Windus, W.A. de Jong
                 "NWChem: a comprehensive and scalable open-source
                  solution for large scale molecular simulations"
                      Comput. Phys. Commun. 181, 1477 (2010)
                           doi:10.1016/j.cpc.2010.04.018

                                      AUTHORS
                                      -------
          E. Apra, E. J. Bylaska, W. A. de Jong, N. Govind, K. Kowalski,
       T. P. Straatsma, M. Valiev, H. J. J. van Dam, D. Wang, T. L. Windus,
        J. Hammond, J. Autschbach, K. Bhaskaran-Nair, J. Brabec, K. Lopata,
       S. A. Fischer, S. Krishnamoorthy, W. Ma, M. Klemm, O. Villa, Y. Chen,
    V. Anisimov, F. Aquino, S. Hirata, M. T. Hackler, V. Konjkov, T. Risthaus,
       M. Malagoli, A. Marenich, A. Otero-de-la-Roza, J. Mullin, P. Nichols,
      R. Peverati, J. Pittner, Y. Zhao, P.-D. Fan, A. Fonari, M. Williamson,
      R. J. Harrison, J. R. Rehr, M. Dupuis, D. Silverstein, D. M. A. Smith,
            J. Nieplocha, V. Tipparaju, M. Krishnan, B. E. Van Kuiken,
        A. Vazquez-Mayagoitia, L. Jensen, M. Swart, Q. Wu, T. Van Voorhis,
     A. A. Auer, M. Nooijen, L. D. Crosby, E. Brown, G. Cisneros, G. I. Fann,
   H. Fruchtl, J. Garza, K. Hirao, R. A. Kendall, J. A. Nichols, K. Tsemekhman,
    K. Wolinski, J. Anchell, D. E. Bernholdt, P. Borowski, T. Clark, D. Clerc,
   H. Dachsel, M. J. O. Deegan, K. Dyall, D. Elwood, E. Glendening, M. Gutowski,
   A. C. Hess, J. Jaffe, B. G. Johnson, J. Ju, R. Kobayashi, R. Kutteh, Z. Lin,
   R. Littlefield, X. Long, B. Meng, T. Nakajima, S. Niu, L. Pollack, M. Rosing,
   K. Glaesemann, G. Sandrone, M. Stave, H. Taylor, G. Thomas, J. H. van Lenthe,
                               A. T. Wong, Z. Zhang.

 Total times  cpu:        0.6s     wall:        0.6s
